utils/src/requests/article/create-article.dto.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, IsOptional, IsString, IsUUID } from 'class-validator';
+import { validationMessages } from '../../validation.util';
+
+export class CreateArticleDto {
```

Comment:
Follow the naming style
```suggestion
export class CreateArticleDTO {
```